scripts/update-ruby.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+if [ "$VERBOSE" = 1 ]; then
+    set -x
+fi
+
+case `sed --help 2>&1` in
+  *GNU*) sed_i () { sed -i "$@"; };;
+  *) sed_i () { sed -i '' "$@"; };;
+esac
+
+SCRIPTS=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+ROOT=$(dirname $SCRIPTS)
```

Comment:
nit: https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
this was copied exactly from `process-podspecs.sh`, but I'll fix in this new file :-)